web/src/cypress/integration/a11y.ts

```diff
@@ -0,0 +1,72 @@
+import { testScreen } from '../support'
+
+function testA11y(): void {
+  describe('no detectable a11y violations', () => {
+    before(() => {
+      // create one of everything, all associated with each other so that every list will be populated
+      cy.createService().then((service) => {
+        cy.setScheduleTarget().then((scheduleTarget) => {
+          cy.createEPStep({
+            epID: service.epID,
+            targets: [{ id: scheduleTarget.scheduleID, type: 'schedule' }],
+          }).then(() => {
+            cy.createManyAlerts(3, { serviceID: service.id })
+          })
+        })
+      })
+    })
+
+    function testRoute(
+      testName: string,
+      route: string,
+      waitForPageLoad: () => void,
+    ): void {
+      it(testName, () => {
+        cy.visit(route)
+        cy.injectAxe()
+        waitForPageLoad()
+        cy.checkA11y(undefined, {
+          includedImpacts: ['critical'], // only report and assert for critical impact items
+        })
```

Comment:
What is the default ruleset, and do we need to specify it here?

Reply:
the default ruleset has everything turned on, including:

- wcag2a
- wcag2aa
- wcag2aaa
- wcag21a
- wcag21aa
- wcag21aaa
- best-practice
- wcag***
- ACT
- section508
- section508.*.*
- experimental
- cat.*

from our discussion though we only want to include WCAG for now:
- wcag2a
- wcag2aa
- wcag2aaa
- wcag21a
- wcag21aa
- wcag21aaa
- best-practice
- wcag***

I'll make these changes 👍